src/pip/_internal/models/link.py

```diff
@@ -179,16 +179,21 @@ def is_wheel(self):
         # type: () -> bool
         return self.ext == WHEEL_EXTENSION
 
+    @property
+    def is_vcs(self):
+        # type: () -> bool
+        from pip._internal.vcs import vcs
+
+        return self.scheme in vcs.all_schemes
+
     @property
     def is_artifact(self):
         # type: () -> bool
         """
         Determines if this points to an actual artifact (e.g. a tarball) or if
         it points to an "abstract" thing like a path or a VCS location.
         """
-        from pip._internal.vcs import vcs
-
-        if self.scheme in vcs.all_schemes:
+        if self.is_vcs:
```

Comment:
Actually, this can just be:

```python
return not self.is_vcs
```

Reply:
Yes of course, but reading the docstring I had the feeling it was meant to evolve to cover other cases. That is also the reason I refrained of implementing `is_vcs` as `not is_artifact`. What do you  think?

Reply:
It's always possible things can change in the future, but we can cross that bridge when we get to it! :)

Reply:
FWIW, I do agree the current docstring is confusing..

Reply:
Okay, done and squashed.

Reply:
Great! 👍